### ISSUE TITLE
Feature request: programmatically open editor panel

### ISSUE THREAD

**Issue by cqnykamp** [Doenet/DoenetML#1081]
## What

Add a React prop to `<DoenetEditor>` that controls the active tab of the problems panels.

## Why

On the website, we need to guide users directly to the accessibility violations.
